ppr-ui/src/components/common/PidNumber.vue

```diff
@@ -89,6 +95,17 @@ export default defineComponent({
       })
     })
 
+    /** Handle pasted event into Pid Input Fields **/
+    const parsePaste = (event: any): void => {
+      // Capture pasted text from event and clean spaces/special chars
+      const pasteInput = event.clipboardData.getData('text').replace(/[^A-Z\d]+/ig, '')
```

Comment:
Nice implementation! 
Do we need to check that length is 9 before pasting?

Reply:
Wasn't really specified, will discuss with Tracey during UX. Allowing a copy paste of over 9 chars and taking the first 9 seems like a better UX than just preventing the paste.
But we will see!